src/api/cf.nim

```diff
@@ -26,12 +27,20 @@ const
   ## base url of the forgesvc endpoint
   modsBaseUrl = "https://addons-ecs.forgesvc.net/api/v2"
 
+
+converter getDependencies(json: JsonNode): int =
```

Comment:
In Nim, a `converter` can be implicitly used to convert between different data types - for example, if a function expects a `CfModFile` and a `JsonNode` is given instead, the converter `toCfModFile` will be implicitly used.

To me, it makes sense to use converters when converting json to/from complex data types since they often have only one "correct" json representation, but not when converting to primitive data types - now, every time a function expects an `int` parameter and receives a `jsonNode`, `getDependencies` is called implicitly, which could create a lot of errors in the future.

Simply converting this to a `proc` should do the trick.